lua/entities/gmod_tardis/modules/sh_doors.lua

```diff
@@ -99,6 +99,15 @@ if SERVER then
 		end
 	end)
 
+	ENT:AddHook("ToggleDoorReal", "classic_doors_collision", function(self,open)
+		if self.metadata.EnableClassicDoors ~= true then return end
+
+		local int_classic_door=TARDIS:GetPart(self.interior,"intdoor")
+		if IsValid(int_classic_door) then
+			int_classic_door:SetCollide(not open, true)
```

Comment:
What's the purpose of this for? In the hell bent one I've been testing this with in-game, you can walk through the interior doors now (not sure if you could before to be fair) when the doors are open. I'm just it's fine just more of a curiosity

Reply:
It's the way portals work
If you can go through normally, you get to the portal exit
If after getting out of the portal you get stuck, you get teleported to Fallback pos, which is okay but not as smooth/precise

You can test it by setting the fallback to a different side of the interior
alt+click would take you to fallback
getting stuck while teleporting would take you to fallback, as it used to happen 
now, the interior doors don't have the collision => you don't get teleported to fallback, you just come out

Reply:
Basically, classic doors used to always teleport you to fallback

Reply:
Ah okay, gotcha